src/Components/Admins/index.js

```diff
@@ -1,9 +1,96 @@
+import { useEffect, useState } from 'react';
 import styles from './admins.module.css';
+import Modal from '../Admins/Modal';
+import Error from '../Admins/Error';
 
 function Admins() {
```

Comment:
I know it came like this, but can we make it an arrow function instead?
```suggestion
const Admins = () => {
```